### PR TITLE
ft(S3C-3484): Add Sensision and extend warp10 tunables

### PR DESCRIPTION
**Pull request by tmacro**
- Adds static.tokens from the federation image
- Adds a disabled by default sensision metrics collector
- Exposes some new tunables for warp10
- Adds some missing flags used in the upstream warp10 init

### CODE REVIEW

**Review by jonathan-gramain — APPROVED** [review]

File: images/warp10/s6/cont-init.d/20-install-config [hunk 1]
```diff
@@ -1,9 +1,14 @@
...
     if [ ! -f "$WARP10_DATA_DIR/conf/$name" ]; then
         cp "$path" "$WARP10_DATA_DIR/conf/$name"
         echo "Copied $name to $WARP10_DATA_DIR/conf/$name"
     fi
-done
\ No newline at end of file
+done
+
+echo "Installing sensision config"
+cp ${SENSISION_HOME}/templates/sensision.template $SENSISION_DATA_DIR/conf/sensision.conf
```
Comment (miniscruff) [inline images/warp10/s6/cont-init.d/20-install-config]: just this one var doesn't have braces, not sure if it's intentional or not.

```suggestion
cp ${SENSISION_HOME}/templates/sensision.template ${SENSISION_DATA_DIR}/conf/sensision.conf
```

File: images/warp10/static.tokens [hunk 2]
```diff
@@ -0,0 +1,9 @@
+token.write.0.name=writeTokenStatic
+token.write.0.producer=42424242-4242-4242-4242-424242424242
```
Comment (miniscruff) [inline images/warp10/static.tokens:2]: is this a test file or example file? Or does all 42's actually work?

Comment (tmacro) [reply images/warp10/static.tokens:2]: I just reused upstream's from here https://github.com/senx/warp10-docker/blob/master/predictible-tokens-for-ci/ci.tokens
AFAIK that's a valid, if highly unlikely, uuid.

Comment (miniscruff) [reply images/warp10/static.tokens:2]: oh are these just for CI purposes?

**Review by miniscruff — APPROVED** [review]

File: images/warp10/static.tokens (hunk 2 above)
Comment (tmacro) [reply images/warp10/static.tokens:2]: not only, we use a static token for deployment as well. Different Dockerfile though.

**Pull request merged**